README.md

```diff
@@ -3,16 +3,27 @@ Example CNF
 
 [![PR Validation](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml/badge.svg)](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml)
 
-Example CNF is an OpenShift workload to exercice an SRIOV setup.
+Example CNF is an OpenShift workload to exercice an SRIOV setup, such as the following:
 
 ![Schema](documentation/schema.png)
 
 It is providing the following operators:
 
-* testpmd-operator
 * trex-operator
+    * It provides TRex Traffic Generator, decomposed in two components: a TRex server, deployed as `trexconfig-<x>` pod, which takes care of deploying and configuring a TRex server instance, and a TRex application, deplayed as `trex-app` job, that starts TRex server, generating traffic towards the system under test.
+    * In `trexconfig-<x>` logs, you can see the trex statistics printed periodically. The summary of the test execution can be seen at the end of the `trex-app` job logs.
+    * In our case, as we will see later on, the `trexconfig-<x>` pod has two interfaces connected to the same SRIOV network.
 * testpmd-lb-operator
+    * Its main component is a modified TestPMD instance, implementing a custom load balancing forwarding module, which is eventually used to perform load balancing between the ports of the deployed pod, called `loadbalancer-<x>`.
+    * This pod is composed by two containers: `loadbalancer`, which performs the load balancing forwarding, and `listener`, an auxiliary module that is listening to the CNFAppMac component created by the cnf-app-mac-operator to retrieve the MAC addresses of the TestPMD instances launched by the testpmd-operator (i.e. the CNF Applications), then serving this information to the `loadbalancer` container.
+    * To see the TestPMD statistics printed periodically for this module, you can rely on `loadbalancer` container logs.
```

Comment:
"you can rely on `loadbalancer` container logs" - we could probably name logfile here. Since we have both loadbalancer and loadbalancer-lb, it's not really clear what exactly to look for

Reply:
I'll rewrite this, but the only log related to LB, apart from controller-manager, is loadbalancer. Loadbalancer-lb does not exist